### ISSUE TITLE
NHS number change - clarify what happens to past records

### ISSUE THREAD

**Issue by fallax** [genderkit/genderkit#252]
We received this via social media:

"On your website about changing gender and name info with the NHS, in the section about being assigned a new NHS number, you state "This means that records from before your change in gender marker are kept separately, to protect your privacy."

This is incorrect. The correct procedure is that your records from your old NHS number are copied to your new NHS number to ensure continuation of care. It would be very dangerous for people with certain conditions to lose their entire medical history/records."

We should check this - ideally with each devolved NHS body if necessary - to make sure we're listing the right information on the page.

**Comment by jlphackett** [comment]
Outcome of discussion on Discord:
- Wording re: NHS systems and gender neutral markers is misleading, maybe reword away from stating "most" NHS systems have other markers, and rephrase it that it could be worthwhile inquiring if your GP has any neutral markers they can use.
- Scrap the bit about records being kept separate, as it's likely incorrect, at least for England. Keep issue open for now in case we turn up something in further research.
- Wales GPOne link is dead, redirects to the homepage of a new site; needs updating
- Current Scottish link also just goes to homepage, https://www.nss.nhs.scot/medical-services/patient-registration-and-medical-records/how-to-change-patient-details/ may be more useful

**Comment by fallax** [comment]
Anyone planning on doing this work - have a chat to me please before trying to update the Wales info
